tests/eval_files/trio232.py

```diff
@@ -7,6 +7,11 @@
 
 async def file_text(f: io.TextIOWrapper):
     f.read()  # TRIO232: 4, 'read', 'f'
+    f.readlines()  # TRIO232: 4, 'readlines', 'f'
+
+    # there might be non-sync calls on TextIOWrappers? - but it will currently trigger
+    # on all calls
+    f.anything()  # TRIO232: 4, 'anything', 'f'
```

Comment:
I'm pretty sure `open`&`close` are still no-go though, so only way to trigger this would be opening a file in a sync context, sending around the `TextIOWrapper` as a parameter inside your async stuff, and then close it once back in a sync context. Which seems like a huge stretch if the only thing you care about is seekable/tell/isatty.
But this might definitely be much more relevant with other objects.

Reply:
Fair enough, and I definitely meant the "might"! I agree that this doesn't seem like it would be much of an improvement in practice, so let's not bother.